proto/babylonlabs/babylon/v1beta1/genesis.proto

```diff
@@ -15,4 +15,6 @@ message GenesisState {
 
   Params params = 1
       [ (gogoproto.nullable) = false, (amino.dont_omitempty) = true ];
-}
+
+  BSNContracts bsn_contracts = 2 [ (gogoproto.nullable) = true ];
```

Comment:
If it's a part of genesis, then this means the contract addresses won't be updatable. I wonder if there exists any scenario where the BSN needs to update this field, for example

- The integration software releases a new major version with more contracts,
- The BSN wants to replace the set of contracts with a new set

So it might be better to maintain those addresses in a separate KV store